database/mssql/scripts/dbscripts/PSP_PIMS_S29_00/Alter Down/51_DML_PIMS_ROLE.sql

```diff
@@ -0,0 +1,18 @@
+UPDATE PIMS_ROLE SET IS_DISABLED = 0, CONCURRENCY_CONTROL_NUMBER = CONCURRENCY_CONTROL_NUMBER + 1 WHERE NAME IN ('Organization Administrator', 'Undetermined', 'Real Estate Manager', 'Real Estate Analyst')
+GO
+
+DECLARE @functional BIGINT;
+DECLARE @readOnly BIGINT;
+DECLARE @rem BIGINT;
+DECLARE @rea BIGINT;
+SELECT @functional = ROLE_ID FROM PIMS_ROLE WHERE NAME = 'Functional';
+SELECT @readOnly = ROLE_ID FROM PIMS_ROLE WHERE NAME = 'Read Only';
+SELECT @rem = ROLE_ID FROM PIMS_ROLE WHERE NAME = 'Real Estate Manager';
+SELECT @rea = ROLE_ID FROM PIMS_ROLE WHERE NAME = 'Real Estate Analyst';
+
+UPDATE PIMS_USER_ROLE SET ROLE_ID = @rea, CONCURRENCY_CONTROL_NUMBER = CONCURRENCY_CONTROL_NUMBER + 1 WHERE ROLE_ID = @functional;
```

Comment:
migrate the real estate agent and real estate manager roles to the roles defined in psp-1057.